src/openfermion/transforms/_conversion.py

```diff
@@ -171,3 +172,59 @@ def get_fermion_operator(interaction_operator):
                                             coefficient)
 
     return fermion_operator
+
+
+def get_molecular_data(interaction_operator,
+                       geometry=None, basis=None, multiplicity=None,
+                       n_electrons=None, reduce_spin=True,
+                       data_directory=None):
+    """Output a MolecularData object generated from an InteractionOperator
+
+    Args:
+        interaction_operator(InteractionOperator): two-body interaction
+            operator defining the "molecular interaction" to be simulated.
+        geometry(string or list of atoms):
+        basis(string):
```

Comment:
Do write something, even if it is obvious. Don't just leave empty.

Reply:
Oops didn't see that